### PR TITLE
Update to v1.28.2 and go to v1.20.8

### PR DESCRIPTION
**Pull request by johnatasr**
#### User-Facing Change ####

```release-note

```

### CODE REVIEW

**Review by brandond — CHANGES_REQUESTED** [review]
```
Unexpected go version go1.21.1 linux/amd64 - Kubernetes v1.28.2 should be built with go version go1.20.8
```

File: .github/workflows/integration.yaml [hunk 1]
```diff
@@ -48,7 +48,7 @@ jobs:
     - name: Install Go
       uses: actions/setup-go@v4
       with: 
-        go-version: '1.20.7'
+        go-version: '1.21.1'
```
Comment (briandowns) [inline .github/workflows/integration.yaml:51]: Only master is using 1.21 right now. THis should be 1.20.8.

**Review by briandowns — APPROVED** [review]

**Review by brandond — APPROVED** [review]
lgtm; make sure to squash when merging.

**Comment by Silvest89** [comment]
@johnatasr Why did you close this? :paintbrush:

**Pull request closed without merging**